agenda/presentation/src/main/java/com/gkp/agenda/presentation/home/components/AgendaListItem.kt

```diff
@@ -0,0 +1,292 @@
+                AgendaItemCircle(
+                    modifier = Modifier,
+                    borderColor = titleColor,
+                    isDone = agendaItem is AgendaItem.Task && agendaItem.isDone
+                )
+                Spacer(modifier = Modifier.width(16.dp))
+                Column {
+                    Text(
+                        text = agendaItem.title,
+                        style = MaterialTheme.typography.titleLarge.copy(
+                            fontWeight = FontWeight.W600,
+                            color = titleColor
+                        ),
+                        overflow = TextOverflow.Ellipsis,
+                        maxLines = 1,
+                        textDecoration = if (agendaItem is AgendaItem.Task && agendaItem.isDone)
+                            TextDecoration.LineThrough
+                        else
+                            null
+                    )
+                    Spacer(modifier = Modifier.height(16.dp))
+                    Text(
+                        text = agendaItem.description?:"",
+                        style = MaterialTheme.typography.bodyMedium.copy(
+                            color = descriptionAndDateColor,
+                            fontWeight = FontWeight.W300
+                        ),
+                        overflow = TextOverflow.Ellipsis,
+                        maxLines = 1
+                    )
+                }
+            }
+            Box(modifier = Modifier.align(Alignment.TopEnd)) {
+                IconButton(
+                    onClick = {
+                        showAgendaItemDropDownMenu = true
+                    }
+                ) {
+                    Icon(
+                        imageVector = ImageVector.vectorResource(R.drawable.ic_group),
+                        contentDescription = null,
+                        tint = titleColor
+                    )
+                }
+                AgendaItemDropDownMenu(
+                    modifier = Modifier,
+                    expanded = showAgendaItemDropDownMenu,
+                    onDismissRequest = {
+                        showAgendaItemDropDownMenu = false
+                    },
+                    onAgendaDropMenuItemClick = onAgendaDropMenuItemClick
+                )
+            }
+
+            Text(
+                modifier = Modifier.align(Alignment.BottomEnd),
+                text = agendaItem.formattedDateTime,
+                style = MaterialTheme.typography.bodyMedium.copy(
+                    color = descriptionAndDateColor,
+                    fontWeight = FontWeight.W300
+                )
+            )
+
+        }
+    }
+}
+
+val agendaItemDropDownMenuTextIdList = listOf(
+    R.string.open,
+    R.string.edit,
+    R.string.delete
+)
+@Composable
+fun AgendaItemDropDownMenu(
+    modifier: Modifier = Modifier,
+    expanded: Boolean,
+    onDismissRequest: () -> Unit,
+    onAgendaDropMenuItemClick: (Int) -> Unit
+) {
```

Comment:
Could it not make more sense to pass these items into your `AgendaItemDropDownMenu` as a parameter? Something like

```
omposable
fun AgendaItemDropDownMenu(
    menuItems: List<String>,
    modifier: Modifier = Modifier,
    expanded: Boolean,
    onDismissRequest: () -> Unit,
    onAgendaDropMenuItemClick: (Int) -> Unit
) {
    // …
    menuItems.forEachIndexed { … }
 }
```